sample-apps/java-events/src/main/java/example/HandlerSQS.java

```diff
@@ -0,0 +1,34 @@
+package example;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.SQSEvent;
+import com.amazonaws.services.lambda.runtime.events.SQSEvent.SQSMessage;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+// Handler value: example.HandlerSQS
+public class HandlerSQS implements RequestHandler<SQSEvent, String>{
+  private static final Logger logger = LoggerFactory.getLogger(HandlerSQS.class);
+  Gson gson = new GsonBuilder().setPrettyPrinting().create();
+  @Override
+  public String handleRequest(SQSEvent event, Context context)
+  {
+    String response = new String("200 OK");
+    // log execution details
```

Comment:
I don't it's helpful to log all this information on each invoke.

Reply:
I am able to strip the function down to just this:

```
// Handler value: example.HandlerSQS
public class HandlerSQS implements RequestHandler<SQSEvent, Void>{
  private static final Logger logger = LoggerFactory.getLogger(HandlerSQS.class);
  @Override
  public Void handleRequest(SQSEvent event, Context context)
  {
    // process event
    for(SQSMessage msg : event.getRecords()){
      logger.info(msg.getBody());
    }
    return null;
  }
}
```

Reply:
Yes :)